test/gtest/common/test_ucc.cc

```diff
@@ -247,6 +247,14 @@ UccJob::UccJob(int _n_procs, ucc_job_ctx_mode_t _ctx_mode, ucc_job_env_t vars) :
 {
     ucc_job_env_t env_bkp;
     char *var;
+
+    /* NCCL TL is disabled since it currently can not support non-blocking
+       team creation. */
+    vars.push_back({"UCC_TL_NCCL_TUNE", "0"});
```

Comment:
is it a temporary fix to disable NCCL also? is there way to going forward fix to testing if non-blocking team create not support

Reply:
it is temporary until we figure out how to run NCCL from single process (not sure if that is possible at all). Gtest is not that well suited for that case